### PR TITLE
chore: add Python support updates to the README

### PR DESCRIPTION
**Pull request by P403n1x87**
Future releases of this package drop support for versions of Python earlier than 3.6. Also inform that the latest version to support Python 3.5 is 0.12.0 and add a link to the Python 2 backport.

### CODE REVIEW

**Comment by MatthieuDartiailh** [comment]
Thanks. I do not know how long you plan to maintain your Python 2 fork so could you rephrase your sentence to make it clear that this project does not make any guarantee about yours. Something like the following would be sufficient: " For Python 2 support, you can have a look at "

**Comment by P403n1x87** [comment]
@MatthieuDartiailh Thanks. Given that Python 2 is dead now, I do not foresee a big maintenance burden from a Python 2-only library. It might just be the occasional bug fix, so hopefully, it'll be manageable and there won't be the need to prepare too many releases.
